### ISSUE TITLE
[BUG] Commented lines too dark to read, and selected comments and whitespace are near-invisible

### ISSUE THREAD

**Issue by eksortso** [51gn3d/vscode-signed-dark-pro#1]
Commented text appears in dark gray upon black. This makes it nearly impossible to read without having to un-comment the code. Selecting the comment makes it even worse to read, and on top of that, whitespace characters cannot be seen clearly when `editor:renderWhitespace` is set to `"selection"`.

Would it be possible to render comments and whitespce in such a way that they can be read without resorting to other themes or to external text viewers?

Unselected code looks like this:
![image](https://user-images.githubusercontent.com/1184444/91084084-6c135a80-e619-11ea-9650-e8c7a3e8656a.png)

Selected code looks like this:
![image](https://user-images.githubusercontent.com/1184444/91084895-a6c9c280-e61a-11ea-9c4d-545007fdce4e.png)

**Comment by eksortso** [comment]
Just wanted to add: I do love this theme aside from this problem, and Victor Mono looks great. I'm complaining, yeah, but it comes from the heart.

**Comment by 51gn3d** [comment]
Thank you...i appreciate it very much

**Comment by 51gn3d** [comment]
Let me look at it immediately

**Comment by eksortso** [comment]
If it helps, I was able to include this in `settings.json` to make the comments more readable. This makes them bright enough to read but not so bright as to be distracting, at least to me. It also works when the comment text is selected.

```json
    "editor.tokenColorCustomizations": {
        "[Signed Dark Pro]": {
            "comments": "#777777"
        }
    },
```

**Comment by 51gn3d** [comment]
I just published Signed Dark Pro v1.2.0 and this issue is addressed. Check it out and maybe give me feedback so that i can close this issue.

**Comment by eksortso** [comment]
@51gn3d I like what you've done. The selection highlighting is a whole lot better. It's all working now! Thank you!
